### ISSUE TITLE
Mark as Deprecated or Move to CCG

### ISSUE THREAD

**Issue by OR13** [w3c/vc-data-integrity#9]
See also: https://github.com/w3c-dvcg/ld-signatures/issues/29

And https://github.com/w3c-dvcg/ld-proofs/issues/5

This documentation needs to be consolidates such that its clear that proofs should look like:

```
{
  "@context": ...,
 ...,
  "proof": {
    "type": "Ed25519Signature2018",
    "created": "2020-01-20T21:38:31Z",
    "verificationMethod": "did:example:123#key-0",
    "jws": "eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..fhRwfF6xZ--Ie1MO6fLLZXbojKQuD6ADKREaEIAp_mAMZIy3HlfWLf0yMUwxU8i2YblhB-T8RLaEepQLfiqcCQ",
    "proofPurpose": "assertionMethod"
  }
}
``` 

Nobody should be constructing proofs with "signature" properties, warnings should be thrown when processing them.

**Comment by dmitrizagidulin** [comment]
Note - this repo has now been moved to the CCG github org.

**Comment by dmitrizagidulin** [comment]
Addressed by PR https://github.com/w3c-ccg/ld-proofs/pull/10

**Comment by msporny** [comment]
This issue was addressed by PR #10 a while ago, closing.
